lte/protos/session_manager.proto

```diff
@@ -90,14 +90,31 @@ message RatSpecificContext {
   }
 }
 
+message TeidSet {
```

Comment:
Let's place any comments in the line above 
```
// Obsoelte to be cleared once AMF code clean up happens
uint32 teid = 1;
```
See [here](https://docs.magmacore.org/docs/next/contributing/contribute_conventions#grpc) for grpc/proto style guidelines. 

(Also obsolete spelling)

Reply:
Yes. I corrected all.